File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.py
lst = list(map(str, input().split('-')))
number = []
for char in lst:
  a = list(map(int, char.split('+')))
  number.append(sum(a))

result = number[0]
for i in range(1, len(number)):
  result -= number[i]

print(result)

File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let arr = input[0].split("-");
let plus_lst = [];

for (let a of arr) {
  let sumValue = a
    .split("+")
    .map(Number)
    .reduce((a, b) => a + b);
  plus_lst.push(sumValue);
}

let result = 0;
for (let num of plus_lst) {
  result -= num;
}
console.log(result + plus_lst[0] * 2);
